### PR TITLE
initial work

### PR DESCRIPTION
**Pull request by AyeshaIftikhar**
I have made the changes in the code. Kindly review it and tell me about the changes if any.
Thank you

### CODE REVIEW

File: lib/screens/welcome_screens/company/components/applicant_view.dart [hunk 1]
```diff
@@ -305,13 +305,14 @@ class _ApplicantViewState extends State<ApplicantView> {
                                       fontSize: 18.0,
                                     ),),
                                     SizedBox(width: 5.0,),
-                                    CountdownTimer(
-                                      endTime: (_getEndTime() / 1000).floor(),
-                                      daysSymbol: Text("days"),
-                                      hoursSymbol: Text("hrs "),
-                                      minSymbol: Text("min "),
-                                      secSymbol: Text("sec"),
-                                    ),
```
Comment (hamzadevc) [inline lib/screens/welcome_screens/company/components/applicant_view.dart:314]: Why removing timer?

File: lib/screens/welcome_screens/employee/components/viewJobDetails.dart [hunk 2]
```diff
@@ -281,13 +281,13 @@ class _ViewJobState extends State<ViewJob> {
                               fontSize: 18.0,
                             ),),
                             SizedBox(width: 5.0,),
-                            CountdownTimer(
-                              endTime: (_getEndTime() / 1000).floor(),
-                              daysSymbol: Text("days"),
-                              hoursSymbol: Text("hrs "),
-                              minSymbol: Text("min "),
-                              secSymbol: Text("sec"),
-                            ),
```
Comment (hamzadevc) [inline lib/screens/welcome_screens/employee/components/viewJobDetails.dart]: Why removing timer?

**Review by hamzadevc — COMMENTED** [review]
Please discuss comments I have added.

File: lib/screens/welcome_screens/company/components/applicant_view.dart (hunk 1 above)
Comment (AyeshaIftikhar) [reply lib/screens/welcome_screens/company/components/applicant_view.dart:314]: wasn't working

File: lib/screens/welcome_screens/employee/components/viewJobDetails.dart (hunk 2 above)
Comment (AyeshaIftikhar) [reply lib/screens/welcome_screens/employee/components/viewJobDetails.dart]: wasn't working

**Review by hamzadevc — APPROVED** [review]

**Pull request merged**